frontend/templates/project_index/template.html

```diff
@@ -50,18 +68,13 @@ <h1 class="title">Projects</h1>
                   </span>
                 </div>
 
-                <transition-group
-                  tag="ul"
-                  class="filter-list"
-                  v-bind:css="false"
-                  v-on:before-enter="filtersBeforeEnter"
-                  v-on:enter="filtersEnter"
-                  v-on:before-leave="filtersBeforeLeave"
-                  v-on:leave="filtersLeave"
-                >
-                  <li v-for="(status, index) in statusesWithCount" v-bind:data-index="index" :key="status" v-if="statusesFilterOpen" :class="status.count === 0 ? 'empty' : ''">
+                <transition-group tag="ul" class="filter-list" v-bind:css="false" v-on:before-enter="filtersBeforeEnter"
```

Comment:
Seems your ran some formatter, this makes it hard to see what is different. The search bar addition should not have changed anything below line 40.

Reply:
Yes, there are not any changes. Only formatting.